scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#268.sql

```diff
@@ -0,0 +1,53 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+-- 1. Clean up animal_release table by deleting animal_release records with no corresponding surgery
+DELETE FROM animal_release ar
+WHERE NOT EXISTS (SELECT release_id FROM surgery s WHERE s.release_id = ar.id);
+
+---- Test results
+-- SELECT * FROM animal_release ar
+-- WHERE NOT EXISTS (SELECT release_id FROM surgery s WHERE s.release_id = ar.id);
+
+-- 2. When capture_location is null, assign value of release_location
+UPDATE animal_release
+SET capture_location = release_location
+WHERE capture_location IS NULL;
```

Comment:
Sure, that'd be another way to do it but doing so would be extremely tedious as we're talking about over 250 records.